ui-charts/src/speedSpaceChart/types.ts

```diff
@@ -1,21 +1,21 @@
-type ElectricalProfileValues = {
+export type ElectricalProfileValues = {
```

Comment:
I'm not sure I understand why this is required. This is only used from `types.ts`, and only `Data` is exported in our public API.

Reply:
added it in speedSpaceChart/index.ts